website/spec/Connections.md

```diff
@@ -173,7 +178,7 @@ Non-Null wrapper around a custom scalar that serializes as a String.
 Whatever type this field returns will be referred to as the *cursor type*
 in the rest of this spec.
 
-The result of this field is considered opaque by Relay, but will be passed
+The result of this field is considered opaque by the spec, but will be passed
```

Comment:
I wonder if this could also be intended as:

```suggestion
The result of this field should be considered opaque by the client, but will be passed
```

🤔 Not sure

Reply:
I've not got a strong opinion either way on this

Reply:
The point is often made [or I'm just making this up] that clients should treat these cursors opaque, hence my thought that maybe by “Relay” they really meant “client”.